### PR TITLE
Updates to Overview/Introduction, fix spelling errors

### PR DESCRIPTION
**Pull request by mkavulich**
- Update Overview for new supported suites and schemes, including links to Scientific Documentation
 - Updates to Introduction to start standardizing the various formats in the doc (bolt, ital, etc.)
 - Fix spelling errors across document

### CODE REVIEW

**Review by climbfuji — APPROVED** [review]

File: CCPPtechnical/source/Overview.rst [hunk 1]
```diff
@@ -81,66 +101,68 @@ and https://dtcenter.org/community-code/common-community-physics-package-ccpp).
...
-(RRFS) and differs from the GSD_v1 suite by not using parameterized convection and employing the
-Noah-Multiparameterization (MP) LSM.
+.. table:: *Suites supported in the CCPP*
+
+   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
+   |                     | :bi:`Operational`| :gbi:`Developmental`                                                                 |
+   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
+   | Physics Suite       | GFS_v16          | :g:`GFS_v17_p8`  | :g:`RAP`       |:g:`RRFS_v1beta`| :g:`WoFS`      | :g:`HRRR`      |
+   +=====================+==================+==================+================+================+================+================+
+   | **Supported hosts** | **SCM/MRW/SRW**  | :g:`**SCM/MRW**` | :g:`**SCM**`   |:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|
```
Comment (ligiabernardet) [inline CCPPtechnical/source/Overview.rst:118]: GFSv16 is not supported with MRW
```suggestion
   | **Supported hosts** | **SCM/SRW**  | :g:`**SCM/MRW**` | :g:`**SCM**`   |:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|
```

Comment (ligiabernardet) [inline CCPPtechnical/source/Overview.rst:118]: There is some issue with the formatting of this line (double asterisks). See https://mike-k-tech-doc.readthedocs.io/en/latest/Overview.html

File: CCPPtechnical/source/Overview.rst [hunk 2]
```diff
@@ -81,66 +101,68 @@ and https://dtcenter.org/community-code/common-community-physics-package-ccpp).
...
+   | Ozone               | NRL 2015         | :g:`NRL 2015`    | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  |
+   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
+   | Strat H\ :sub:`2`\ O| NRL 2015         | :g:`NRL 2015`    | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  |
+   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
+   | Ocean               | NSST             | :g:`NSST`        | :g:`NSST`      | :g:`NSST`      | :g:`NSST`      | :g:`NSST`      |
+   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
+
+Only the suites that are currently supported in the CCPP are listed in the table. Currently all supported suites use the 2015 Navy Research Laboratory (NRL) `ozone <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__o_z_p_h_y_s.html>`_ and `stratospheric water vapor <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__h2_o_p_h_y_s.html>`_ schemes,
```
Comment (ligiabernardet) [inline CCPPtechnical/source/Overview.rst:143]: The future/permanent location for the CCPP v6 SciDoc will be https://dtcenter.ucar.edu/GMTB/v6.0.0, without the "p" in the end. Do you want to fix this now (which will result in broken links) or later?

File: CCPPtechnical/source/Overview.rst [hunk 3]
```diff
@@ -81,66 +101,68 @@ and https://dtcenter.org/community-code/common-community-physics-package-ccpp).
...
+
+* The RAP scheme is similar to the operational Rapid Refresh (RAP) model physics package, and features Thompson microphysics, `Mellor-Yamada-Nakanishi-Niino (MYNN) EDMF <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_m_y_n_n_e_d_m_f.html>`_ PBL, `Grell-Freitas (GF) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_c_u__g_f.html>`_ deep convection and shallow convection schemes, RRTMG radiation, `MYNN surface layer (SFL) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_s_f_c__m_y_n_n_s_f_l.html>`_ scheme, `Global Systems Laboratory (GSL) drag <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s_drag_suite.html>`_ gravity wave drag scheme, and the `Rapid Update Cycle (RUC) Land Surface Model <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_r_u_c_l_s_m.html>`_.
```
Comment (ligiabernardet) [inline CCPPtechnical/source/Overview.rst:159]: I am trying to fix drag appearing twice in the same sentence
```suggestion
* The RAP scheme is similar to the operational Rapid Refresh (RAP) model physics package, and features Thompson microphysics, `Mellor-Yamada-Nakanishi-Niino (MYNN) EDMF <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_m_y_n_n_e_d_m_f.html>`_ PBL, `Grell-Freitas (GF) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_c_u__g_f.html>`_ deep convection and shallow convection schemes, RRTMG radiation, `MYNN surface layer (SFL) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_s_f_c__m_y_n_n_s_f_l.html>`_ scheme, `Global Systems Laboratory (GSL) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s_drag_suite.html>`_ gravity wave drag scheme, and the `Rapid Update Cycle (RUC) Land Surface Model <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_r_u_c_l_s_m.html>`_.
```

File: CCPPtechnical/source/Overview.rst [hunk 4]
```diff
@@ -162,15 +184,15 @@ upgrade with the capability to build the code using Python 3 (previously only Py
...
 .. [#] As of this writing, the CCPP has been validated with two host models: the CCPP
     SCM and the atmospheric component of
     NOAA’s Unified Forecast System (UFS) (hereafter the UFS Atmosphere) that utilizes
-    the Finite-Volume Cubed Sphere (FV3) dycore.  The CCPP can be utilized both with the
-    global and limited-area configurations of the UFS Atmosphere. CCPP v4.1.0 is the latest
-    release compatible with the global UFS Medium-Range Weather Application. CCPP v5.0.0 is only
-    compatible with the limited-area UFS Short-Range Weather Application.  The CCPP
+    the Finite-Volume Cubed Sphere (FV3) dynamical core.  The CCPP can be utilized both with the
+    global and limited-area configurations of the UFS Atmosphere. CCPP v6.0.0 is the latest
+    release compatible with the global UFS Medium-Range Weather Application and the
```
Comment (ligiabernardet) [inline CCPPtechnical/source/Overview.rst:194]:
```suggestion
    release compatible with the global UFS MRW Application and the
```

File: CCPPtechnical/source/Overview.rst [hunk 5]
```diff
@@ -162,15 +184,15 @@ upgrade with the capability to build the code using Python 3 (previously only Py
...
+    limited-area UFS Short-Range Weather Application.  The CCPP
```
Comment (ligiabernardet) [inline CCPPtechnical/source/Overview.rst:195]:
```suggestion
    limited-area UFS SRW Application.  The CCPP
```

**Review by ligiabernardet — CHANGES_REQUESTED** [review]
Thank you for the great bug fixes, updates, and enhancements.

In particular, thank you for including a good explanation of operational and developmental suites. I am not sure what to do with the concept you introduced of "experimental". You said _*experimental* suites are included in the `ccpp_physics` repository_ but this is not correct because ccpp-physics does not have suites (only schemes). My suggestion is to remove the "experimental" concept altogether, since all suites in the release are either operational or developmental.  However, I get the feeling what you are trying to say is that there are schemes in the repo (and in the released code) that are unsupported. If so, perhaps you can use the word unsupported and refer to the schemes but not to the suites.

Please insert this bold text in the Introduction chapter. 
(it replaced suite GFSv15plus). **CCPP v4.0 was the first release supported for use with the UFS Weather Model, more specifically as part of the UFS Medium-Range Weather (MRW) Application.** The CCPP v4.1 release,

File: CCPPtechnical/source/Overview.rst [hunk 6]
```diff
@@ -81,66 +101,68 @@ and https://dtcenter.org/community-code/common-community-physics-package-ccpp).
 The table below lists all parameterizations supported in CCPP public releases and the
 `CCPP Scientific Documentation <https://dtcenter.ucar.edu/GMTB/v5.0.0/sci_doc>`_
```
[the remark this answers is not in the document]
Comment (mkavulich) [reply CCPPtechnical/source/Overview.rst]: @ligiabernardet, I am including a bunch of links to the scientific document below as well. For now I have been pointing to the temporary locations as this makes it easier to ensure that the links are correct, and they can be updated all at once when the scientific documentation is published (it is listed in the issue tracking "to-do" tasks). I think that's a better way to go about it, but if you feel strongly I can make the update to the "future" links now.

File: CCPPtechnical/source/Overview.rst (hunk 2 above)
Comment (mkavulich) [reply CCPPtechnical/source/Overview.rst:150]: See my comment above, I'd rather fix it later if that's all right with you.

**Comment by mkavulich** [comment]
@ligiabernardet I believe I have resolved all your suggestions except for the bit about SciDoc links, let me know what you think. I also re-built the temporary readthedocs for easier viewing: https://mike-k-tech-doc.readthedocs.io/en/latest/Overview.html

File: CCPPtechnical/source/Overview.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (ligiabernardet) [reply CCPPtechnical/source/Overview.rst]: OK, this is a reasonable way forward

**Review by ligiabernardet — APPROVED** [review]
HI Mike, Thanks for all the updates. I went already and approved the PR but there was one change I requested but could not find in the resulting HTML.  So, if you could this one before merging it would be great.

Please insert this bold text in the Introduction chapter.
(it replaced suite GFSv15plus). **CCPP v4.0 was the first release supported for use with the UFS Weather Model, more specifically as part of the UFS Medium-Range Weather (MRW) Application.** The CCPP v4.1 release,

**Comment by mkavulich** [comment]
Sorry about missing that first comment! Added it.

**Pull request merged**